src/mut/morphology.cpp

```diff
@@ -357,12 +371,13 @@ void Morphology::write(const std::string& filename) {
         extension += my_tolower(c);
 
     if (extension == ".h5")
-        writer::h5(clean, filename);
+        writer::h5(*this, filename);
```

Comment:
Eventually have:
```suggestion
        _informIfUnifurcations();
        writer::h5(*this, filename);
```
to make people aware of unifurcations - spread the word :)

Reply:
let's also add an issue for that.